src/geo/leaflet/leaflet-cartodb-layer-group-view.js

```diff
@@ -43,18 +46,25 @@ var findContainerPoint = function (map, o) {
   return point;
 };
 
-var LeafletCartoDBLayerGroupView = function (layerModel, leafletMap) {
+var LeafletCartoDBLayerGroupView = function (layerModel, leafletMap, mapModel) {
   var self = this;
   LeafletLayerView.apply(this, arguments);
-  CartoDBLayerGroupViewBase.call(this, layerModel, leafletMap);
+  CartoDBLayerGroupViewBase.apply(this, arguments);
 
-  this.leafletLayer.on('load', function (e) {
+  this.mapModel = mapModel;
+
+  this.leafletLayer.on('load', function () {
     self.trigger('load');
   });
 
-  this.leafletLayer.on('loading', function (e) {
+  this.leafletLayer.on('loading', function () {
     self.trigger('loading');
   });
+
+  this.leafletLayer.on('tileerror', function (layer) {
+    layer.tile.src = TILE_ERROR_IMAGE;
+    self.mapModel.addError({ type: C.WINDSHAFT_ERRORS.TILE });
```

Comment:
I think I already mentioned this in a previous CR, but I think it might make sense so I'd like to bring it up again... 

Both `src/geo/leaflet/leaflet-cartodb-layer-group-view.js` and `src/geo/gmaps/gmaps-cartodb-layer-group-view.js` use the same model: https://github.com/CartoDB/cartodb.js/blob/v4/src/geo/cartodb-layer-group.js. This model (virtually) "groups" layers of type "CartoDB". Since the tile errors that we're handling in this PR are related to this group of layers, I think it'd make more sense to "store" those error in the `CartoDBLayerGroup` (i.e: I'd move `addError` and the `errors` collection from `MapModel` to `LayerGroupModel`.

I'm asking because we're changing CartoDB.js right now and the new feature that we're introducing will allow developers to use `src/geo/leaflet/leaflet-cartodb-layer-group-view.js` and `src/geo/gmaps/gmaps-cartodb-layer-group-view.js` without the need of a `MapModel` (developers will work directly with the Leaflet or GoogleMaps map). We'll need to expose tile/limit errors in there too.

It should be a pretty straightforward change. We'd do `self.model.addError(...)` instead of `self.mapModel.addError(...)` here.

Reply:
Do you think we should keep a collection of errors? Or it's better just to trigger the `error:limit` / `error:tile` and that's it?